doc/userguide/configuration/exception-policies.rst

```diff
@@ -0,0 +1,101 @@
+.. _exception policies:
+
+Exception Policies
+==================
+
+Suricata has a set of configuration variables to indicate what should the engine
+do when certain exception conditions, such as hitting a memcap, are reached.
+
+They are called Exception Policies and are configurable via suricata.yaml. If
+enabled, the engine will call them when it  reaches exception states.
+
+For developers or for researching purposes, there are also simulation options
+exposed in debug mode and passed via command-line. These exist to force or
+simulate failures or errors and understand Suri behavior under such conditions.
+
+Exception Policies
+------------------
+
+Exception policies are implemented for:
+
+.. list-table:: Exception Policies configuration variables
+   :widths: 20, 18, 62
+   :header-rows: 1
+
+   * - Config setting
+     - Policy variable
+     - Expected behavior
+   * - stream.memcap
+     - memcap-policy
```

Comment:
is it clear enough that the setting here is `stream.memcap-policy`?

Reply:
Considering what's written a few lines below:
"To change any of those, check the corresponding section in the suricata.yaml file. "
And how these variables are presented in the yaml file, I think it should be ok. I will add a reference to the suricata.yaml documentation, too, since it also shows these settings...